lib/fastlane/actions/ensure_xcode_version.rb

```diff
@@ -0,0 +1,59 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      ENSURE_XCODE_VERSION_CUSTOM_VALUE = :ENSURE_XCODE_VERSION_CUSTOM_VALUE
+    end
+
+    class EnsureXcodeVersionAction < Action
+      def self.run(params)
+        required_version = params[:version]
+
+        selected_version = sh "xcversion selected | head -1 | xargs echo -n"
+
+        versions_match = selected_version == "Xcode #{required_version}"
+
+        raise "Selected Xcode version doesn't match your requirement.\nExpected: Xcode #{required_version}\nActual: #{selected_version}" unless versions_match
```

Comment:
Maybe also add instructions on how to fix it? What would you do if that doesn't match? Run `xcode-install` and then `xcode-select`?